docs/testing-types/index.md

```diff
@@ -3,6 +3,23 @@ title: Testing types
 layout: default
 nav_order: 4
 has_children: true
+has_toc: false
 ---
 
-# Testing types applied for AWS
\ No newline at end of file
+# Testing types applied for AWS
+
+The following list is a high-level description of each testing type relevant to this guide. Each testing type plays a crucial role in ensuring the quality, reliability, and performance of applications in the AWS cloud environment.
+{: .fs-6 .fw-300 }
+
+To learn more about each type and best practices for implementation, we recommend visiting their respective pages. This guide provides a comprehensive overview of testing practices tailored specifically to the AWS ecosystem.
+
+- **Manual Testing**: Manual testing involves human testers executing test cases and exploring the application's functionality to identify defects. While automated testing is preferred for its efficiency, manual testing remains valuable, especially for user experience validation, usability testing, and ad-hoc testing in the AWS cloud environment.
```

Comment:
Compared to other items in this list, I feel like this does not really fit here, or at least not in its current form.
It speaks of usability and UX validation, which is mostly the same regardless if it is deployed to AWS or to an on-prem server.

The other items in this list speak about testing types which test at different levels of abstraction, but they can be done both manually and in an automated way. 

I would suggest deleting this bullet point, and just having some text at line 15, like `For the below testing types, automated testing is preferred, but manual testing remains valuable, especially for ad-hoc testing in the AWS cloud environment and hard to automate test cases.`